### PR TITLE
Add TestGetTemplates integration test

### PR DESCRIPTION
**Pull request by chenilim**
#### Summary
Add an integration test for retrieving and using built-in templates

### CODE REVIEW

File: server/integrationtests/board_test.go [hunk 1]
```diff
@@ -1268,3 +1269,53 @@ func TestDeleteMember(t *testing.T) {
...
+		th := SetupTestHelper(t).InitBasic()
+		defer th.TearDown()
+
+		teamID := "my-team-id"
+		rBoards, resp := th.Client.GetTemplatesForTeam("0")
+		th.CheckOK(resp)
+		require.NotNil(t, rBoards)
+		require.GreaterOrEqual(t, len(rBoards), 6)
+
+		fmt.Printf("\n\n")
```
Comment (wiggin77) [inline server/integrationtests/board_test.go:1284]: Can we use `t.Log` or `t.Logf` for these?  It captures the output in order with the rest of the log output, and also allows us to selectively output it or not based on test success/failure.

Comment (chenilim) [reply server/integrationtests/board_test.go]: Thanks! Fixed.

**Review by harshilsharma63 — APPROVED** [review]

**Pull request merged**